Doc/library/string.rst

```diff
@@ -236,12 +236,14 @@ does an index lookup using :func:`__getitem__`.
 
 Some simple format string examples::
 
-   "First, thou shalt count to {0}"  # References first positional argument
-   "Bring me a {}"                   # Implicitly references the first positional argument
-   "From {} to {}"                   # Same as "From {0} to {1}"
-   "My quest is {name}"              # References keyword argument 'name'
-   "Weight in tons {0.weight}"       # 'weight' attribute of first positional arg
-   "Units destroyed: {players[0]}"   # First element of keyword argument 'players'.
+   "First, thou shalt count to {0}"    # References first positional argument
+   "Then, thou shalt count to {{{1}}}" # References first positional argument and adds escaped curly
+                                       # brace: ("...to {1}")
```

Comment:
I don't think this is a great example. The "... to {1}" isn't equivalent to the other "Same as" example. I think you need to say it's the value of the second positional argument, enclosed in curly braces.